main.go

```
@@ -79,6 +80,7 @@ func init() {
 	utilruntime.Must(controllerv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(dwv1.AddToScheme(scheme))
 	utilruntime.Must(dwv2.AddToScheme(scheme))
+	utilruntime.Must(gwapiv1.AddToScheme(scheme))
```

golangci-lint (2.12.2)

[error] SA1019: gwapiv1.AddToScheme is deprecated: use Install instead 

(staticcheck)